src/renderer/renderScene.ts

```diff
@@ -129,7 +134,13 @@ export function renderScene(
     translateContextToZoom();
     context.translate(sceneState.scrollX, sceneState.scrollY);
     context.translate(selectionElement.x, selectionElement.y);
```

Comment:
And here you are not. I think we should maintain this API consistent.

If I'm not wrong, you should be able to leave the translates for the visible elements rendering and remove them from the 2 branches of `renderElement` that I'm commenting next.